.github/workflows/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+    attr_accessor :date, :person, :book
+  
+    def initialize(date, person, book)
+      @date = date
+      @person = person
+      @book = book
+    end
```

Comment:
- [ ] Let's modify the Rental class to add itself to the rentals arrays of the classes.

  ```
  def initialize(date, person, book)
    @date = date

    @person = person
    person.rentals << self

    @book = book
    book.rentals << self
  end
  ```

Reply:
done